packages/mongodb-artifact-generator/src/docs-meta-descriptions/generateMetaDescription.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  assertEnvVars,
+  CORE_OPENAI_CHAT_COMPLETION_ENV_VARS,
+  OpenAIClient,
+} from "mongodb-rag-core";
+import { RunLogger } from "../runlogger";
+import { stripIndents } from "common-tags";
+import { readdirSync, readFileSync } from "fs";
+import path from "path";
+import { assistantMessage, systemMessage, userMessage } from "../chat";
+import { z } from "zod";
+
+export type MetaDescription = string;
+
+export type MakeGenerateMetaDescription = {
+  openAiClient: OpenAIClient;
+  logger?: RunLogger;
+};
+
+const systemPrompt = stripIndents`
+  Your task is to generate a concise, informative description that accurately reflects the content of a given text page.
+
+  This brief summary will be used as a meta description for the page across various services, including search engines.
+
+  The meta description should conform to the following style guide:
+
+  - The meta description should be one single sentence
+    - no character limit
+    - roughly 150 characters is a good target length
+  - The meta description should start with a verb
+  - The meta description should not repeat the page title verbatim
+  - The meta description should not repeat low value or assumed terms like
+    - Don't mention "MongoDB" in the description of a MongoDB docs page.
+    - Don't mention "documentation" in the description of a docs page.
+
+  Respond only with the generated description.
+`;
+
+export function makeGenerateMetaDescription({
```

Comment:
light unit testing on basic i/o pls. 

also consider evals when productionizing this feature